### PR TITLE
Trim less text in overview tables by using CSS instead of custom code

### PR DESCRIPTION
**Pull request by okurz**
Alternative to https://github.com/os-autoinst/openQA/pull/1090

### CODE REVIEW

**Comment by okurz** [comment]
before:
openqa_table_names_old_ellipsis

after:
openqa_table_names_ellipsis

so not much visual difference, loosing less characters, cleaner code.

**Comment by okurz** [comment]
rebased. Let's see if travis checks succeed

**Comment by okurz** [comment]
Hi, I'm getting the following errors in travis and need some help to understand, I don't think they are caused by my changes:

```
[19:11:23] ./t/27-websockets.t ....................... 1/?     # [DEBUG] Worker Boooo not seen since 0 seconds

[ERROR] Received unexpected WS message "$VAR1 = '';

 from worker 1

[ERROR] Unknown property received from worker CODE(0x9deb848)

[19:11:23] ./t/27-websockets.t ....................... ok     2116 ms ( 0.00 usr  0.00 sys +  6.17 cusr  0.15 csys =  6.32 CPU)

[19:11:25] ./t/28-logging.t .......................... ok      234 ms ( 0.00 usr  0.00 sys +  0.54 cusr  0.03 csys =  0.57 CPU)

[19:11:29] ./t/29-openqa_commands.t .................. 1/? fake started with pid 7435

[19:11:29] ./t/29-openqa_commands.t .................. 2/? Uncaught exception from user code:

	org.freedesktop.DBus.Error.AccessDenied: Connection ":1.8" is not allowed to own the service "org.opensuse.openqa.Scheduler" due to security policies in the configuration file

	Test::Builder::subtest('Test::Builder=HASH(0x27523f8)', 'run', 'CODE(0x26d2ee0)') called at /home/travis/perl5/perlbrew/perls/5.18/lib/site_perl/5.18.4/Test/More.pm line 771

	Test::More::subtest('run', 'CODE(0x26d2ee0)') called at ./t/29-openqa_commands.t line 54

Uncaught exception from user code:

	Unable to load "assetpack.def". at /home/travis/perl_modules/lib/perl5/Mojolicious/Plugin/AssetPack.pm line 185.

	Test::Builder::subtest('Test::Builder=HASH(0x27523f8)', 'run', 'CODE(0x26d2ee0)') called at /home/travis/perl5/perlbrew/perls/5.18/lib/site_perl/5.18.4/Test/More.pm line 771

	Test::More::subtest('run', 'CODE(0x26d2ee0)') called at ./t/29-openqa_commands.t line 54

Uncaught exception from user code:

	DBIx::Class::Storage::DBI::catch {...} (): DBI Connection failed: DBI connect('dbname=/var/lib/openqa/db/db.sqlite','',...) failed: unable to open database file at /home/travis/perl_modules/lib/perl5/DBIx/Class/Storage/DBI.pm line 1517. at lib/OpenQA/Schema.pm line 114

	Test::Builder::subtest('Test::Builder=HASH(0x27523f8)', 'run', 'CODE(0x26d2ee0)') called at /home/travis/perl5/perlbrew/perls/5.18/lib/site_perl/5.18.4/Test/More.pm line 771

	Test::More::subtest('run', 'CODE(0x26d2ee0)') called at ./t/29-openqa_commands.t line 54

Uncaught exception from user code:

	org.freedesktop.DBus.Error.AccessDenied: Connection ":1.9" is not allowed to own the service "org.opensuse.openqa.Scheduler" due to security policies in the configuration file

	Test::Builder::subtest('Test::Builder=HASH(0x27523f8)', 'run', 'CODE(0x26d2ee0)') called at /home/travis/perl5/perlbrew/perls/5.18/lib/site_perl/5.18.4/Test/More.pm line 771

	Test::More::subtest('run', 'CODE(0x26d2ee0)') called at ./t/29-openqa_commands.t line 54
```

File: templates/test/overview.html.ep [hunk 1]
```diff
@@ -134,5 +135,92 @@
                 %= include 'test/overview_result_table', type => $type, type_results => $type_result, type_archs => $type_archs->{$type}
             % }
         % }
+||||||| merged common ancestors
```
Comment (foursixnine) [inline templates/test/overview.html.ep]: It seems there's a conflict here.

File: templates/test/overview.html.ep [hunk 2]
```diff
@@ -134,5 +135,92 @@
...
+                                % next;
+                            % }
+                            % my $resultid = join('_', $type, $arch, $config);
+                            %= include 'test/tr_job_result', resultid => $resultid, res => $res, state => $state, jobid => $jobid
+                        % }
+                    </tr>
+                % }
+            </tbody>
+        </table>
+=======
```
Comment (foursixnine) [inline templates/test/overview.html.ep]: It seems there's a conflict here.

File: templates/test/overview.html.ep [hunk 3]
```diff
@@ -134,5 +135,92 @@
...
+                                % next;
+                            % }
+                            % my $resultid = join('_', $type, $arch, $config);
+                            %= include 'test/tr_job_result', resultid => $resultid, res => $res, state => $state, jobid => $jobid
+                        % }
+                    </tr>
+                % }
+            </tbody>
+        </table>
+>>>>>>> Trim less text in overview tables by using CSS instead of custom code
```
Comment (foursixnine) [inline templates/test/overview.html.ep]: It seems there's a conflict here.

**Comment by foursixnine** [comment]
As for the problems related to dbus @ https://travis-ci.org/os-autoinst/openQA/jobs/278741175#L811 maybe the dbus session failed to start.

**Comment by mudler** [comment]
That test is explictly leveraging a part of code that will make dbus connection fails, as a dbus session is set up just for fullstack tests ( see https://github.com/os-autoinst/openQA/blob/master/Makefile#L100 ) , the dbus errors reported are sane (as long as the single test end up to 'ok')

File: templates/test/overview.html.ep (hunk 1 above)
Comment (okurz) [reply templates/test/overview.html.ep]: that might explain some errors ;) Updated

**Pull request merged**